src/components/notifications/AppExplorer/index.tsx

```diff
@@ -1,43 +1,35 @@
-import { useContext, useMemo } from 'react'
-import { useSearch } from '../../../utils/hooks'
-import BackButton from '../../general/BackButton'
 import AppCard from './AppCard'
 import './AppExplorer.scss'
-import AppExplorerHeader from './AppExplorerHeader'
 import useNotifyProjects from '../../../utils/hooks/useNotifyProjects'
-import W3iContext from '../../../contexts/W3iContext/context'
+import MobileHeader from '../../layout/MobileHeader'
+import IntroContent from '../../general/IntroContent'
+import Button from '../../general/Button'
+import IntroApps from '../../general/Icon/IntroApps'
 
 const AppExplorer = () => {
-  const { appSearchTerm } = useSearch()
   const projects = useNotifyProjects()
-  const { activeSubscriptions } = useContext(W3iContext)
-
-  const filteredApps = useMemo(
-    () =>
-      projects.filter(app => {
-        if (appSearchTerm) {
-          return (
-            app.name.includes(appSearchTerm) ||
-            app.description.includes(appSearchTerm) ||
-            app.url.includes(appSearchTerm)
-          )
-        }
-
-        const activeSubscriptionAppNames = activeSubscriptions.map(sub => sub.metadata.appDomain)
-        const hostname = new URL(app.url).host
-
-        return !activeSubscriptionAppNames.includes(hostname)
-      }),
-    [appSearchTerm, projects, activeSubscriptions]
-  )
 
   return (
     <div className="AppExplorer">
-      <BackButton backTo="/notifications">Notifications</BackButton>
-      <AppExplorerHeader />
+      <MobileHeader title="Discover" />
+      <IntroContent
+        title="Welcome to Web3Inbox"
+        subtitle="Subscribing to our available apps below to start receiving notifications"
+        button={
+          <Button
```

Comment:
There was some discussion on Slack regarding the fact that we should remove this button. Not sure what was the resolution.

Reply:
Conclusion was to remove/hide: https://walletconnect.slack.com/archives/C044SKFKELR/p1695137955836439

It's a noop anyways atm.